dev-env/bin/daml-sdk-head

```diff
@@ -27,6 +27,9 @@ for opt in "$@" ; do
     "--profiling")
       PROFILING=1
       ;;
+    "--skip-jars")
+      SKIP_JARS=1;
+      ;;
```

Comment:
Nice. Could you add `SKIP_JARS=0` before the `for` loop to match the handling of the other command line args?